src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/SnowplowEvent.scala

```diff
@@ -17,34 +21,26 @@ trait SnowplowEvent {
 object SnowplowEvent {
 
   case class UnstructEvent(data: Option[SelfDescribingData[Json]]) extends AnyVal
+  val unstructEventUri = SchemaKey("com.snowplowanalytics.snowplow", "unstruct_event", "jsonschema", SchemaVer.Full(1, 0, 0))
 
   case class Contexts(data: List[SelfDescribingData[Json]]) extends AnyVal
+  val contextsUri = SchemaKey("com.snowplowanalytics.snowplow", "contexts", "jsonschema", SchemaVer.Full(1, 0, 0))
 
   implicit final val contextsCirceEncoder: io.circe.Encoder[Contexts] =
     Encoder.instance { contexts: Contexts =>
       if (contexts.data.isEmpty) List[String]().asJson
       else JsonObject(
-        ("schema", "iglu:com.snowplowanalytics.snowplow/contexts/jsonschema/1-0-0".asJson),
-        ("data", contexts.data.map { selfdesc: SelfDescribingData[Json] =>
-          JsonObject(
-            ("schema", selfdesc.schema.toSchemaUri.asJson),
-            ("data", selfdesc.data)
-          )
-        }.asJson)
+        ("schema", contextsUri.toSchemaUri.asJson),
+        ("data", contexts.data.map { selfdesc => selfdesc.asJson }.asJson)
```

Comment:
`("data", contexts.data.asJson)` We do have encoders for `SelfDescribingData[Json]`, we do have encoders for `List`.